jenkins/spark-nightly-build.sh

```diff
@@ -120,13 +119,12 @@ distWithReducedPom "install"
 if [[ $SKIP_DEPLOY != 'true' ]]; then
     distWithReducedPom "deploy"
 
-    # this deploy includes 'tools' that is unconditionally built with Spark 3.1.1
+    # this deploy sub modules except dist that is unconditionally built with Spark 3.1.1
```

Comment:
minor typo
```sugestion
    # this deploys submodules except dist that is unconditionally built with Spark 3.1.1
```

Reply:
ok, let me file a follow-up PR to fix it.